coalib/core/Graphs.py

```diff
@@ -0,0 +1,59 @@
+from coalib.core.CircularDependencyError import CircularDependencyError
+
+
+def traverse_graph(start_nodes, get_successive_nodes,
```

Comment:
it's actually useful, see https://github.com/coala/coala/pull/3523/commits/5b3b87c96f262769946d1e3617170c915323b49e

Each edge I traverse in the dependency graph is a dependency relation I feed the `DependencyTracker` with.